cmd/main.go

```diff
@@ -379,6 +379,15 @@ func main() {
 
 	publicRegistries := map[string]bool{}
 	validations.ParsePublicRegistries(publicRegistries, Options.ValidationsConfig.PublicRegistries)
+	if mirrorRegistriesBuilder.IsMirrorRegistriesConfigured() {
+		mirrorRegistries, err := mirrorRegistriesBuilder.ExtractLocationMirrorDataFromRegistries()
+		if err != nil {
+			log.WithError(err).Warnf("failed to parse mirror registries provided to assisted-service, check your mirror registry config map")
+		} else {
+			validations.ParseMirrorRegistries(publicRegistries, mirrorRegistries)
```

Comment:
I can see that `validations.ParseMirrorRegistries` will parse into publicRegistries. 
What happens if any of the mirror registries fail to parse, should we handle this somehow?

Reply:
Good idea, I added a log message inside the parse mirror registries function. What do you think?